python/cross-region-codeartifact/README.md

```diff
@@ -0,0 +1,92 @@
+
+# CrossRegionArtifact Project
+
+## Introduction
+
+Customers utilize AWS CodeArtifact to store and distribute software packages , currently AWS CodeArtifact is only available in 13 regions listed [here](https://aws.amazon.com/codeartifact/faq/).
+
+In this guide we will will provide a demo on how to utilize AWS CodePipeline via AWS CodeBuild to build and publish an artifact to CodeArtifact and then consume the built artifact in another CodePipeline to deploy a lambda and layer built from the custom package hosted in a cross region CodeArtifact repository.
```

Comment:
Rewrite this in the form of "In this example, you will use AWS CodePipeline via AWS CodeBuild to build and publish and artifact..."

Include information on how customers can see the effects of the example as it does its thing. 

Include information on the usage of CDK Nags.